_posts/en/2024-10-09-wallet-integration-guide.md

```diff
@@ -0,0 +1,590 @@
+100kvB non-TRUC limit). This restricted limit still supports larger batch
+payments as well. If a wallet has no choice but to spend an unconfirmed input,
+if that input comes from a TRUC transaction, this new transaction would need to
+be below 1kvB.
+
+This would allow an RBFing wallet confidence that receivers of bitcoin would not
+be able to create arbitrary chains of transactions off of the payment, pinning
+the user. Incoming payments could also be robustly bumped via an up to 1kvB
+spend of the incoming deposit output.
+
+### Coinjoins
+
+In the [coinjoin][topic coinjoin] scenario where privacy is the focus but the coinjoin is not
+attempting to be covert, TRUC transactions for the coinjoin itself may be
+worthwhile. The coinjoin may have insufficient feerate for inclusion in the
+blockchain requiring a small bump of a participants’ coins.
+
+Along with TRUC transactions, a P2A output could be added allowing for a
+segregated wallet like a watchtower to pay for the transaction fees alone.
+
+If other participants spend their unconfirmed outputs, TRUC sibling eviction
+will ensure that the other spends can be replaced even if not directly
+conflicting, preserving TRUC topology limits.
+
+Pinning caveat: Participants in the coinjoin may still economically grief the
+transaction by double-spending their own input to the coinjoin, requiring the
+coinjoin to RBF the griefer’s first transaction.
+
+### Lightning Network
+
+Transactions generated in the Lightning Network protocol consist of a few main
+types:
+
+1. Funding transactions: Single-party funded or “dual”-party funded
+transactions to set up the contract. Less time sensitivity to confirm.
+2. Commitment transactions: The transaction that commits to the latest state of the
+payment channel. These transactions are asymmetrical, and currently require a
+“update_fee” message bi-directionally to update how much of the funding output
+value is given to fees. The fees must be enough to propagate the latest version
+of the commitment transaction into miners’ mempools.
+3. HTLC presigned transactions
+
+With 1P1C relay and package RBF, upgrading Bitcoin Core nodes significantly
+increases the security of the Lightning network. Lightning unilateral closes can
+be accomplished with commitment transactions with below mempool minfee feerates,
+or conflicting with another low-fee commitment transaction package that wouldn’t
+be promptly included in a block.
+
+To take the maximal advantage of this upgrade, wallets and backends should
+integrate with the **submitpackage** Bitcoin Core RPC command:
+
+```hack
+bitcoin-cli submitpackage ‘[“<commitment_tx_hex>”, “<anchor_spend_hex>”]’
+```
+
+Wallet implementations should integrate their software with the command using
+the commitment transaction as well as an anchor child spend to ensure inclusion
+into miners’ mempools with the appropriate feerate.
+
+Note: The RPC endpoint supports many-child, single-parent packages, but these
```

Comment:
```suggestion
Note: The RPC endpoint will return success if you submit a many-child, single-parent package, but these
```

Just want to make it a bit more clear that they won't receive any indication that propagation is unlikely to succeed from the RPC if they're dependent on a 1p1c RBF.